docs/reference/configuration/keywords.rst

```diff
@@ -2069,61 +2069,89 @@ Workflow hooks
-        HOOK_WORKFLOW QC_WFLOW1        POST_SIMULATION
-        HOOK_WORKFLOW QC_WFLOW2        POST_SIMULATION
+        - Before the simulations (all forward models for a realization) start using :code:`PRE_SIMULATION`,
+        - after all the simulations have completed using :code:`POST_SIMULATION`,
+        - before the update step using :code:`PRE_UPDATE`
+        - after the update step using :code:`POST_UPDATE` and
+        - only before the first update using :code:`PRE_FIRST_UPDATE`.
 
+        For non interactive algorithms, :code:`PRE_FIRST_UPDATE` is equal to :code:`PRE_UPDATE`.
+        The :code:`POST_SIMULATION` hook is typically used to trigger QC workflows.
 
-    In this example the workflow :code:`initWFLOW` will run after all
-    the simulation directories have been created, just before the
-    forward model is submitted to the queue. The workflow
-    :code:`preUpdateWFLOW` will be run before the update step and
-    :code:`postUpdateWFLOW` will be run after the update step. When
-    all the simulations are complete the two workflows
-    :code:`QC_WFLOW1` and :code:`QC_WFLOW2` will be run.
+        ::
 
-    Observe that the workflows being 'hooked in' with the
-    :code:`HOOK_WORKFLOW` must be loaded with the
-    :code:`LOAD_WORKFLOW` keyword.
+           HOOK_WORKFLOW initWFLOW        PRE_SIMULATION
+           HOOK_WORKFLOW preUpdateWFLOW   PRE_UPDATE
+           HOOK_WORKFLOW postUpdateWFLOW  POST_UPDATE
+           HOOK_WORKFLOW QC_WFLOW1        POST_SIMULATION
+           HOOK_WORKFLOW QC_WFLOW2        POST_SIMULATION
 
-    Currently, :code:`PRE_UPDATE` and :code:`POST_UPDATE` are only
-    available from python.
+        In this example the workflow :code:`initWFLOW` will run after all the
+        simulation directories have been created, just before the forward
+        model is submitted to the queue. The workflow :code:`preUpdateWFLOW`
+        will be run before the update step and :code:`postUpdateWFLOW` will be
+        run after the update step. When all the simulations have completed the
+        two workflows :code:`QC_WFLOW1` and :code:`QC_WFLOW2` will be run.
 
+        Observe that the workflows being 'hooked in' with the
+        :code:`HOOK_WORKFLOW` must be loaded with the :code:`LOAD_WORKFLOW` keyword.
 
 .. _load_workflow:
 .. topic:: LOAD_WORKFLOW
 
-    Load a workflow into ERT.
+        Workflows are loaded with the configuration option :code:`LOAD_WORKFLOW`:
 
+        ::
+
+            LOAD_WORKFLOW  /path/to/workflow/WFLOW1
+            LOAD_WORKFLOW  /path/to/workflow/workflow2  WFLOW2
+
+        The :code:`LOAD_WORKFLOW` takes the path to a workflow file as the first
+        argument. By default the workflow will be labeled with the filename
+        internally in ERT, but you can optionally supply a second extra argument
+        which will be used as the name for the workflow.  Alternatively,
+        you can load a workflow interactively.
```

Comment:
What does interactively mean here?

Reply:
Collectively decided to not follow up on this comment.